.github/workflows/inception-test.yml

```diff
@@ -24,12 +30,28 @@ jobs:
           python -m pip install coverage
           python -m pip install .
 
+      - name: Install datalad
+        if: matrix.extra == 'datalad'
+        run: python -m pip install datalad
+
       - name: Checkout con/tinuous-inception
         uses: actions/checkout@v2
         with:
           repository: con/tinuous-inception
           path: tinuous-inception
 
+      - name: Configure datalad
+        if: matrix.extra == 'datalad'
+        run: |
+          git config --global user.email "tinuous@nil"
+          git config --global user.name "Tinuous Tester"
+
+          sed -i -e 's:{year}/{month}/{day}:{year}//{month}//{day}:' config.yml
+          echo "datalad:" >> config.yml
+          echo "  enabled: true" >> config.yml
+          echo "  cfg_proc: text2git" >> config.yml
```

Comment:
I think you would need to `datalad create -f .` or at least `git init` here, since otherwise DataLad is probably not used?  I do not see coverage report for some reason but I suspect that no datalad logic was triggered since I expected it to fail, since IIRC we require git-annex to be installed for operations, and git-annex isn't installed, so I would have expected it to crash (I could be wrong though ;))

Reply:
[tinuous creates a dataset if one doesn't exist](https://github.com/con/tinuous/blob/89024374b4ed8b45c9d3b7b826f3196e4872b7c3/src/tinuous/__main__.py#L97-L99), and the output from the test run ends with:

```
2021-06-10T15:35:55+0000 [INFO    ] datalad.core.local.save Total: starting
119
2021-06-10T15:35:57+0000 [INFO    ] datalad.core.local.save 
120
2021-06-10T15:35:57+0000 [INFO    ] datalad.core.local.save Total: processed result for /home/runner/work/tinuous/tinuous/tinuous-inception
121
2021-06-10T15:35:57+0000 [INFO    ] datalad.core.local.save Total: done
```

Reply:
The tests now install git-annex.